composer.json

```diff
@@ -22,7 +22,7 @@
         "php-http/discovery": "1.15.0",
         "symfony/dependency-injection": "5.4.16 || 6.0.16 || 6.1.8 || 6.2.0 || 6.2.1",
         "symfony/finder": "3.4.7 || 4.0.7 || 5.4.26 || 6.2.13 || 6.3.2",
-        "symfony/framework-bundle": "4.2.7 || 5.2.6 || 5.4.30",
+        "symfony/framework-bundle": "4.2.7 || 5.2.6 || 5.4.30 - 5.4.31 || 6.3.6 - 6.3.8"
```

Comment:
me missed the comma here 🙈

Reply:
Fixed in https://github.com/contao/conflicts/pull/54

Reply:
woops, sorry